jwk.go

```diff
@@ -779,7 +779,13 @@ func (key rawJSONWebKey) symmetricKey() ([]byte, error) {
 	return key.K.bytes(), nil
 }
 
-func tryJWKS(key interface{}, headers ...Header) interface{} {
+var (
+	// ErrJWKSKidNotFound is returned when a JWKS does not contain a JWK with a
+	// key ID which matches one in the provided tokens headers.
+	ErrJWKSKidNotFound = errors.New("go-jose/go-jose: JWK with matching kid not found in JWKS")
```

Comment:
Nit:

```suggestion
	ErrJWKSKidNotFound = errors.New("go-jose/go-jose: JWK with matching kid not found in JWK Set")
```

[The RFC uses "JWS Set"](https://datatracker.ietf.org/doc/html/rfc7517#section-5) and doesn't seem to abbreviate it to JWKS anywhere.

Reply:
Awesome - I'm not deeply familiar with the spec but this makes sense.